### ISSUE TITLE
Lightbox playing incorrect video with multiple videos in gallery

### ISSUE THREAD

**Issue by Nuuou** [biati-digital/glightbox#187]
**Describe the bug**
Using multiple videos (all Vimeo, not sure if relevant), clicking on a link to open the lightbox results in an incorrect video being played. This is a new issue for me in 3.0.6. Downgrading to 3.0.5 seems to fix this for me.

Related thread: https://github.com/biati-digital/glightbox/issues/182

**Are you able to reproduce the bug in the demo site**
No

**To Reproduce**
1. Go to https://codesandbox.io/s/glightbox-306-bug-v4u66
2. Click either Video 1 or Video 3
3. Notice incorrect video plays

**Expected behavior**
Upon clicking a link, I would expect the corresponding video to play. However, and incorrect video is playing

**Post the code you are using**
https://codesandbox.io/s/glightbox-306-bug-v4u66

**Desktop:**
 - OS: macOS 10.15.7
 - Browser: Chrome, Firefox
 - Version: Chrome 87, Firefox 84

**Comment by biati-digital** [comment]
If i click 1 or 3 i see the correct video https://cln.sh/qlSw38

**Comment by Nuuou** [comment]
Apologies, let me rephrase! I realize that's misleading now.

While you _see_ the correct, video, wait a second and notice that another video is actually playing simultaneously (Video 2).

As the `autoplayVideos` is set to `true` by default, it's almost as if it's autoplaying the second one, which is incorrect. In your recording, you'll notice that the video you click _isn't_ autoplaying, which would be a bug in general right?

**Comment by biati-digital** [comment]
Hi @Nuuou ahhhh thank you for the explanation, yes, that's a bug. i'll try to take a look at this maybe next week, can't promise anything, i really need some time to rest i'm feeling exhausted.

**Comment by puleeno** [comment]
I have the same issue. Please fix it when you have free time. Thank you!!

**Comment by biati-digital** [comment]
This should be fixed in 3.0.7, please try it and let me know if it's working for you

**Comment by didrip** [comment]
Still experiencing this bug with 3.0.7.

**Comment by biati-digital** [comment]
Hi @didrip please provide an example as this should be fixed in the latest version

**Comment by didrip** [comment]
Hi @biati-digital, you were too quick to answer I did not have time to edit!

It is OK now after I removed the `autoplay: true` from plyr `config`. Previously:

```js
    plyr: {
			css: 'https://cdn.plyr.io/3.6.2/plyr.css', // default value
			js: 'https://cdn.plyr.io/3.6.2/plyr.polyfilled.js', // polyfilled version
			ratio: '16:9',
			config: {
				// hideControls: true,
				autoplay: true,
				controls: ['play-large', 'progress', 'mute', 'volume', 'fullscreen'],
				vimeo: { color: 'red' }
			}
```

This config is useless anyway because controls and colors are taken from Vimeo video settings.
